### ISSUE TITLE
errors during start and stop

### ISSUE THREAD

**Issue by mbiebl** [mbentley/docker-omada-controller#10]
hi,

thanks for providing these docker images.

During runtime of the container, I get the following error messages:
```
Exception in thread "Thread-0" java.lang.IllegalArgumentException: URI is not hierarchical
        at java.io.File.<init>(File.java:418)
        at com.tp_link.eap.util.g.a.a(SourceFile:47)
        at com.tp_link.eap.configuration.b.a(SourceFile:58)
        at com.tp_link.eap.util.o.c(SourceFile:443)
        at com.tp_link.eap.util.o.b(SourceFile:434)
        at com.tp_link.eap.start.f.run(SourceFile:35)
startup...
Nov 09, 2019 4:26:56 AM org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 4.3.1.Final
Omada Controller started
ShutdownHook: service stopped.
Exception in thread "Thread-0" java.lang.IllegalArgumentException: URI is not hierarchical
        at java.io.File.<init>(File.java:418)
        at com.tp_link.eap.util.g.a.a(SourceFile:47)
        at com.tp_link.eap.configuration.b.a(SourceFile:58)
        at com.tp_link.eap.util.o.c(SourceFile:443)
        at com.tp_link.eap.util.o.b(SourceFile:434)
        at com.tp_link.eap.start.f.run(SourceFile:35)
startup...
Nov 09, 2019 1:05:55 PM org.hibernate.validator.internal.util.Version <clinit>
INFO: HV000001: Hibernate Validator 4.3.1.Final
Omada Controller started


ShutdownHook: service stopped.
```

**Comment by KYLE-HILL** [comment]
See if this fixes your issue. https://github.com/mbentley/docker-omada-controller/issues/6

**Comment by mbiebl** [comment]
This error only happens when I start the docker container for the first time. After a restart the error is gone.

**Comment by fenix-silver** [comment]
I'm seeing a similar issue after updating to Unraid 6.8.  Couldn't get the container to start at all.  Even tried creating a new instance.  Downgraded back to 6.7.2 and it started right up.

**Comment by mbiebl** [comment]
I no longer see his error with 3.2.6.

**Comment by mbentley** [comment]
@mbiebl - think this is good to close as being fixed in 3.2.6 then?  I haven't paid attention to the logs.

**Comment by mbiebl** [comment]
I think so. At least I haven't seen this error anymore after upgrading to 3.2.6.
